spec/requests/api/alerts_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe "Alerts API" do
+  let(:alert_status)  { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_status2) { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_url) { alerts_url(alert_status.id) }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:alert_definition) { FactoryGirl.create(:miq_alert, :severity => "info") }
+  let(:user) { FactoryGirl.create(:user_with_group) }
+
+  it "forbids access to actions without an appropriate role" do
+    api_basic_authorize
+    run_get(alerts_url)
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "reads all alerts statuses" do
+    api_basic_authorize collection_action_identifier(:alerts, :read, :get)
+    alert_status
+    alert_status2
+    run_get(alerts_url)
+    expect(response).to have_http_status(:ok)
+    expect(response.parsed_body["count"]).to eq(2)
+  end
```

Comment:
Unfortunately that actual connection is only added in https://github.com/ManageIQ/manageiq/pull/13323

That test is the code that was commented out here and I promise to add it back once relevant.
(Did that to separate the patches as requested and the `object.try(:miq_alert_statuses)` is also so we have no problem merging this)